### PR TITLE
Updates from rwr+ renewal on July 2024

### PR DESCRIPTION
**Pull request by Tschis**
Updated with details after starting renewal process for RWR+, in July 2024

### CODE REVIEW

File: pages/work-permit/extension.mdx [hunk 1]
```diff
@@ -34,18 +33,16 @@ To start the renewal process, you will need the following documents:
   - 3 latest payment slips (yours, if you are currently employed)
 - Meldezettel (if you moved since your last application)
 
-If you have a partner, it's important to have the following copies of their
-documents:
+If you have a partner, it i important to have the following copies of their documents:
```
Comment (Potonho) [inline pages/work-permit/extension.mdx]: it is

**Comment by Tschis** [comment]
@brunojppb fyi

File: public/files/rwr_plus_form_20231101_downloaded-july2024.pdf
Comment (brunojppb) [inline public/files/rwr_plus_form_20231101_downloaded-july2024.pdf:1]: This seems to be the old form, like the one I used back then when I did the process for the first time. The one you removed has the new city identity and so on (branding with logo and the right font).

Which leads me to believe that these MA35 offices are not in sync with the "right" or up2date forms available.

Maybe we should include both and state that and let the user choose?

Comment (Tschis) [reply public/files/rwr_plus_form_20231101_downloaded-july2024.pdf:1]: The existing file has on the bottom of its pages: `KF-M35003 - 032021`
The file I uploaded has: `Stand: September 2023`
Which indicates my file is newer.

Additionally, the file is directly downloaded from [Bundesministerium Inneres](https://www.bmi.gv.at/312/60a/start.aspx) (whatever that means 😂), but that sounds like the correct source. Additionally, the link from [Wien.at](https://www.wien.gv.at/amtshelfer/dokumente/aufenthalt/drittstaaten/bescheinigungen/rwr-karte-schluesselkraft-selbstaendig.html) (scroll down to the `Formular` section) also points to the same BMI form.

Comment (Tschis) [reply public/files/rwr_plus_form_20231101_downloaded-july2024.pdf:1]: From which link can we get the current form, today?

Comment (brunojppb) [reply public/files/rwr_plus_form_20231101_downloaded-july2024.pdf:1]: > From which link can we get the current form, today?

I remember that I got it from the [terminvereinbarung aussenstellen website](https://www.wien.gv.at/kontakte/ma35/terminvereinbarung-aussenstellen.html) while I was booking my appointment back then. But tbh, this is so inconsistent that it's likely they rolled back the "new" version of the form. It's a pitta tho, it was so pretty 🤣

Let's roll!

**Review by brunojppb — APPROVED** [review]

**Pull request merged**